### ISSUE TITLE
Setting schema:true breaks validation

### ISSUE THREAD

**Issue by AmarPrabhu** [balderdashy/sails-mongo#326]
If `schema: true` on a model, then
``` 
Mymodel
.create(mydata)
.then(function(modeldata){
    //do foo
})
.catch(function(err){
    if(err){
        console.log(err.code); //returns null
        console.log(err.invalidAttributes); //returns null
    }
}); 
```

setting `schema:false` however returns the error codes and invalid attributes. Am I missing something here? What should be the Model if I want only defined attributes to be inserted, and also get validation errors?

**Comment by sailsbot** [comment]
Thanks for posting, @AmarPrabhu.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!

**Comment by sailsbot** [comment]
Thanks for posting, @AmarPrabhu.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!

**Comment by devinivy** [comment]
Marking this as a question.  May also be a bug.

**Comment by sailsbot** [comment]
Thanks for posting, @AmarPrabhu.  I'm a repo bot-- nice to meet you!

The issue queue in this repo is for verified bugs with documented features.  Unfortunately, we can't leave some other types of issues marked as "open".  This includes bug reports about undocumented features or unofficial plugins, as well as feature requests, questions, and commentary about the core framework. Please review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) for details.

If you're here looking for help and can't find it, visit [StackOverflow](http://stackoverflow.com), our [Google Group](https://groups.google.com/forum/#!forum/sailsjs) or our [chat room](https://gitter.im/balderdashy/sails?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge).  But please do not let this disrupt future conversation in this issue! I am only marking it as "closed" to keep organized.

Thanks so much for your help!

> If I am mistaken, and this issue is about a _critical bug with a documented feature_, please accept my sincerest apologies-- I am but a humble bot working to make GitHub a better place.  If you open a new issue, a member of the core team will take a look ASAP.
